commit 072c5925f867ecf3b1567f898abe7ace6b283079
Author: Kailang Yang <kailang@realtek.com>
Date:   Wed Nov 8 15:28:33 2017 +0800

    ALSA: hda/realtek - Fix ALC275 no sound issue
    
    commit 3aabf94c2d95fe465d5fa8590113d1c1f7d8333d upstream.
    
    Sound works after a cold boot but not after a reboot from windows.
    This patch will solve this issue. This is relation with Class-D power control.
    
    [ The bug was reported in Bugzilla below for Sony VAIO SVS13A1C5E
      -- tiwai]
    
    Bugzilla: https://bugzilla.kernel.org/show_bug.cgi?id=197737
    Signed-off-by: Kailang Yang <kailang@realtek.com>
    Signed-off-by: Takashi Iwai <tiwai@suse.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/sound/pci/hda/patch_realtek.c b/sound/pci/hda/patch_realtek.c
index dce0682c5001..804a5707662a 100644
--- a/sound/pci/hda/patch_realtek.c
+++ b/sound/pci/hda/patch_realtek.c
@@ -341,6 +341,9 @@ static void alc_fill_eapd_coef(struct hda_codec *codec)
 	case 0x10ec0299:
 		alc_update_coef_idx(codec, 0x10, 1<<9, 0);
 		break;
+	case 0x10ec0275:
+		alc_update_coef_idx(codec, 0xe, 0, 1<<0);
+		break;
 	case 0x10ec0293:
 		alc_update_coef_idx(codec, 0xa, 1<<13, 0);
 		break;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1091_072c5925f867ecf3b1567f898abe7ace6b283079.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1091_072c5925f867ecf3b1567f898abe7ace6b283079.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
